package.json

```diff
@@ -21,5 +21,8 @@
     "purescript"
   ],
   "author": "Thimoteus",
-  "license": "MIT"
+  "license": "MIT",
+  "dependencies": {
```

Comment:
I'm not sure if we want to have any `dependencies` here. Should I drop this `pulp` from here?

Reply:
Yeah, if anything it should be a `devDependencies`.